### PR TITLE
VT-652: Allow hapi route config, and turn off auth on healthcheck

### PR DESCRIPTION
**Pull request by BradAtTurner**

### CODE REVIEW

**Comment by BradAtTurner** [comment]
Regarding the workaround we discussed, I tried to find the path to check for '_healthcheck' with the library we are using for authentication, but that info is not available until the hapi 17 upgrade.  So I made this PR- we will need to be able to pass route config in the future anyways.

File: src/hapi.js [hunk 1]
```diff
@@ -16,9 +16,9 @@ function registerRoutes(app, { routes : r }, callback) {
 
     routes.unshift(healthcheckRoute);
```
Comment (BradAtTurner) [inline src/hapi.js]: I was gonna add that same logic to Express, but I looked up the documentation, and extra parameters are assumed to be extra callback functions.

File: src/hapi.js [hunk 2]
```diff
@@ -16,9 +16,9 @@ function registerRoutes(app, { routes : r }, callback) {
```
Comment (BradAtTurner) [inline src/hapi.js]: Note to self: bump version and tag before merging

**Review by rwinbush — APPROVED** [review]

**Comment by adslaton** [comment]
How can I test these changes?

**Comment by adslaton** [comment]
Disregard my comment above.

**Review by adslaton — APPROVED** [review]

**Pull request merged**